README.md

```diff
@@ -1,6 +1,22 @@
 # FairMQ
 
-C++ Message Queuing Library
+C++ Message Queuing Library and Framework
+
+FairMQ was designed to help creating large-scale processing pipelines needed in next-generation High Energy
+Physics Experiments. FairMQ is written in C++ and aims 
+  * to be **easy to use** (hide the details of low level data transport technologies),
+  * to provide a reasonably **efficient data transport** (zero-copy, high throughput) service,
+  * that is **data format agnostic**, and
+  * to provide some of the **basic building blocks** to implement a higher level data processing framework on top.
```

Comment:
Even though you describe this in the following paragraph, I would add also here one more point about the abstract interface that allows implementation with efficient current and future transport technologies.